src/UWP/ArcGISRuntime.UWP.Viewer/Samples/Layers/WmsIdentify/WmsIdentify.xaml.cs

```diff
@@ -8,21 +8,20 @@
 // language governing permissions and limitations under the License.
 
 using Esri.ArcGISRuntime.Data;
-using Esri.ArcGISRuntime.Geometry;
 using Esri.ArcGISRuntime.Mapping;
 using Esri.ArcGISRuntime.Ogc;
 using System;
 using System.Collections.Generic;
 using System.Linq;
-using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml;
```

Comment:
FYI: UWP doesn't have same WebView-DPI-problem here that iOS did. So this WMS sample worked fine all along.

Reply:
I know, but when I updated the other samples to not use callouts you recommended that I do the same for desktop because showing arbitrary/unpredictable HTML in a callout is bad pattern to show.